src/main/java/com/arpnetworking/kairosdb/aggregators/HistogramPercentileAggregator.java

```diff
@@ -74,9 +72,22 @@ public String getAggregatedGroupType(final String groupType) {
         return _dataPointFactory.getGroupType();
     }
 
-
-    @javax.validation.constraints.Min(0)
-    @javax.validation.constraints.Max(1)
+    @NonZero
+    @FeatureProperty(
+            label = "Percentile",
+            description = "Data points returned will be in this percentile.",
+            default_value = "0.1",
```

Comment:
This seems like an odd default value.

Reply:
Yep, but that's the default for the non-histo one.